src/main/groovy/nebula/plugin/publishing/ivy/IvyResolvedDependenciesPlugin.groovy

```diff
@@ -84,25 +45,21 @@ class IvyResolvedDependenciesPlugin extends AbstractResolvedDependenciesPlugin {
         }
     }
 
-    private VersionSelector parseSelector(String version) {
-        def scheme = new DefaultVersionSelectorScheme(new DefaultVersionComparator())
-        def selector = scheme.parseSelector(version)
-        selector
-    }
-
-    private void setVersionConstraint(VersionSelector selector, String version, Node dep, ModuleVersionIdentifier selected) {
-        if (!(selector instanceof ExactVersionSelector)) {
-            //requested dynamic version will be replaced by specific selected
-            dep.@revConstraint = version
-            dep.@rev = selected.version
-        }
-    }
-
-    private void updateReplacedModules(ModuleVersionIdentifier mvid, String group, String name, Node dep) {
-        if (mvid.group != group || mvid.name != name) {
-            dep.@org = mvid.group
-            dep.@name = mvid.name
-            dep.@rev = mvid.version
+    private void verifyFirstDependencyExclusions(Project project) {
```

Comment:
What case is this verification catching? Do we have a test?

Reply:
we do, here is the test: https://github.com/nebula-plugins/nebula-publishing-plugin/blob/95713c49a04ebbad4ff78571687e1e6369614997/src/test/groovy/nebula/plugin/publishing/ivy/IvyResolvedDependenciesPluginIntegrationSpec.groovy#L177

Reply:
Oh I see it was originally in the parent class.

Reply:
yeah! sorry, forgot to mention that!